src/Service/Services/MetadataProviders/CosmosSqlMetadataProvider.cs

```diff
@@ -25,7 +25,9 @@ public class CosmosSqlMetadataProvider : ISqlMetadataProvider
         private readonly RuntimeConfig _runtimeConfig;
         private Dictionary<string, string> _partitionKeyPaths = new();
         private Dictionary<string, string> _graphQLSingularTypeToEntityNameMap = new();
+        private Dictionary<string, List<FieldDefinitionNode>> _graphQLTypeToFieldsMap = new();
         private readonly RuntimeConfigProvider _runtimeConfigProvider;
+        public DocumentNode root;
```

Comment:
nit: public members should be first letter UpperCase 
so `Root`, also could name it more meaningful e.g. `GraphQLSchemaRoot`

Reply:
Updated.